transients_nearby_galaxies/transients_nearby_galaxies.ipynb

```diff
@@ -0,0 +1,1182 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">target_mask[4,4]=True
target_mask[5,4]=True
target_mask[5,5]=True
</pre><p>When I included more pixels, I was able to produce a less noisy final plot that made the supernova "stand out" a bit more</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
<p>I think adding this as an exercise, as you did below, is probably the best idea.</p>